samples/account-linking/csharp/.tours/tab-account-linking.tour

```diff
@@ -0,0 +1,221 @@
+{
+  "$schema": "https://aka.ms/codetour-schema",
+  "title": "Tab-Account-Linking",
+  "steps": [
+    {
+      "title": "Introduction",
+      "description": "# Introduction\r\nThis is a tour of the Tab's usage of the account linking for calling the [Github Repos API](https://docs.github.com/en/rest/reference/repos#list-repositories-for-a-user).\r\n\r\nAt a high level this uses the backend service as a proxy for the /repositories API."
+    },
+    {
+      "file": "Source/wwwroot/index.html",
+      "description": "# The Tab Page\r\nThis is an extremely minimal tab implementation. In practice you will likely build a more complex single-page application but to keep this focused on the core functionality this is built using pure js & the [Microsoft Teams client sdk](https://docs.microsoft.com/en-us/javascript/api/overview/msteams-client?view=msteams-client-js-latest).\r\n\r\nThis app just dumps the JSON payload of the Repositories API into the body of the tab\r\n\r\n![Screenshot of json payload of GitHub repositories displayed in a Teams Personal tab](./Docs/Images/TabScreenshot.jpg)",
```

Comment:
This image doesn't show when testing in vscode. Might need to use a path to the root of the repo, I'm not sure